digitaltwins/e2e/digitaltwin_commands.js

```diff
@@ -92,14 +92,14 @@ describe('Digital Twin Invoke Command', function () {
         }
       });
     });
-    digitalTwinClient.addInterfaceInstance(testInterfaceInstance);
+    digitalTwinClient.addComponent(testComponent);
```

Comment:
>addComponent [](start = 22, length = 12)

We're gonna have to fix these e2e tests since they still are using register and addComponent should be addComponents